.travis.yml

```diff
@@ -1,17 +1,12 @@
 language: python
 python: 2.7
 env:
-  - PLONE_VERSION=4.2
```

Comment:
you're only testing effectively for Plone 5.0; are you dropping compatibility with previous versions?